rusqlite_migration/src/asynch.rs

```diff
@@ -144,12 +144,29 @@ impl AsyncMigrations {
     /// ```
     #[allow(clippy::missing_errors_doc)]
     pub async fn to_version(&self, async_conn: &mut AsyncConnection, version: usize) -> Result<()> {
-        let m = self.migrations.clone();
+        let m = Arc::clone(&self.migrations);
         async_conn
             .call(move |conn| Ok(m.to_version(conn, version)))
             .await?
     }
 
+    /// Proxy version of the same method in the
+    /// [Migrations](crate::Migrations::latest_schema_version) struct.
+    pub fn latest_schema_version(&self) -> usize {
```

Comment:
As I mentionned in the issue, I'm not convinced it's worth exposing both this method and the boolean one. The boolean one seems simpler to use, so I would suggest we don't make that one public.

Reply:
Actually, there are use cases for both (see https://github.com/cljoly/rusqlite_migration/issues/170#issuecomment-2343979134), I’m fine with exposing both.